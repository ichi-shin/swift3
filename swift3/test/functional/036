#!/bin/bash
# Copyright(c)2014 NTT corp. All Rights Reserved.
# DB1-1-1-7

. ./common

S3USER=tester

_s3_get / -D - | _filter_curl xml
