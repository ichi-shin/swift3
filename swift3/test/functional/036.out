QA output created by 036
> s3curl --id tester -- -X GET -D - http://SWIFT_HOST/... 200
HTTP/1.1 200 OK
Content-Length: LENGTH
Content-Type: application/xml
Date: DATE
x-amz-id-2: TXID
x-amz-request-id: TXID
X-Trans-Id: TXID

<?xml version='1.0' encoding='UTF-8'?>
<ListAllMyBucketsResult xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
  <Owner>
    <ID>TESTER</ID>
    <DisplayName>TESTER</DisplayName>
  </Owner>
  <Buckets/>
</ListAllMyBucketsResult>
